wikipendium/wiki/static/js/script.js

```diff
@@ -107,6 +107,15 @@ $(function(){
         $(tabId).addClass('active');
     });
 
+    $(window).on('hashchange', function(e) {
+      // Impressively, this ensures scrolling doesn't break (as much) in Safari
+      $('.toc').toggleClass('yolo');
+    });
+
+    $('.section1 h1').click(function(e) {
+      $(this).parent('.section1').toggleClass('open');
+    });
+
```

Comment:
Could you separate this into its own commit? Since it's a bit of a hack. Makes it easier to revert if safari fixes its issues later on.